python/desc/sims/GCRCatSimInterface/generate_lensed_hosts_sne.py

```diff
@@ -0,0 +1,346 @@
+#matplotlib inline
+#config InlineBackend.figure_format = 'retina'
+
+import numpy as np
+import os
+import pylab as pl
+import subprocess as sp
+import astropy.io.fits as pyfits
+import pandas as pd
+import scipy.special as ss
+import om10_lensing_equations as ole
+
+outdir = "./outputs/"
+data_dir = os.path.join(os.path.abspath("../../../../"),'data')
+twinkles_data_dir = os.path.join(os.environ['TWINKLES_DIR'], 'data')
+
+def random_location(Reff_src, qs, phs, ns):
+    """Sample a random (x, y) location from the surface brightness
+    profile of the galaxy."""
+
+    phs_rad = np.deg2rad(phs-90)
+
+    bn = ss.gammaincinv(2. * ns, 0.5)
+    z = np.random.random_sample()
+    x = ss.gammaincinv(2. * ns, z)
+    R = (x / bn)**ns * Reff_src
+    theta = np.random.random() * 2 * np.pi
+    
+    xp, yp = R * np.cos(theta), R * np.sin(theta)
+    xt = xp * np.sqrt(qs)
+    yt = yp / np.sqrt(qs)
+    dx, dy = np.linalg.solve([[np.cos(phs_rad), np.sin(phs_rad)],
+                             [-np.sin(phs_rad), np.cos(phs_rad)]],
+                             [xt, yt])
+    return dx, dy
+
+
+def check_random_locations():
+    
```

Comment:
Please add docstrings for the methods in this script.  It will make it easier to maintain this code in the future.